frontend/pulse3d/src/serviceWorker.js

```diff
@@ -92,7 +98,9 @@ const modifyRequest = async (req, url) => {
   const modifiedReq = new Request(getUrl(url), {
     headers,
     body:
-      req.method === "POST" ? JSON.stringify(await req.clone().json()) : null,
+      req.method === "POST" || req.method === "PUT"
```

Comment:
Did you try `body: JSON.stringify(await req.clone().json()) || null`? I think Luci said that it works. It's a little cleaner than adding a bunch of checks on the method

Reply:
`body: JSON.stringify(await req.clone().json()) || null ` gives multiple errors